orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -1332,16 +1332,16 @@ IMPLEMENT_GRADIENT_BUILDER(GetFastGeluGradient) {
 }
 
 IMPLEMENT_GRADIENT_BUILDER(GetLayerNormalizationGradient) {
-  if (GetGradientGraphConfiguration().use_invertible_layernorm_grad) {
+  if (IsTensorStashed(I(0).name)) {
```

Comment:
>I(0) [](http://example.com/codeflow?start=22&length=4)

I(0) will trigger the stashing... 
So this if check will always return true.

Reply:
O(0) stashed then use invertible.